src/api/DclData.ts

```diff
@@ -35,6 +36,30 @@ export type TransparencyData = {
   teams: Team[]
 }
 
+type Grants = {
```

Comment:
how about we call this `TransparencyGrant`?

Reply:
I like it!